lib/omnibus/builder.rb

```diff
@@ -635,6 +635,37 @@ def sync(source, destination, options = {})
     end
     expose :sync
 
+
+    def update_config_guess(target = '', version = 'master', options = {})
+      destination = File.join(software.project_dir, target)
+
+      c_g_data = {
+        :locked_version => version,
+        :locked_source => {
+          :git=>"http://git.savannah.gnu.org/r/config.git"
+        },
+        :source_type => 'git',
+        :described_version => version
+      }
+
+      config_guess_manifest = ManifestEntry.new('config_guess', c_g_data)
+
+      config_guess_dir = File.join(Config.source_dir, "config_guess-#{version}")
+      config_guess_fetcher = GitFetcher.new(config_guess_manifest, config_guess_dir, destination)
```

Comment:
So I agree we need the `update_config_guess` DSL function in Omnibus core but the fetching of a dependency (even just build time) up in the bowls of Omnibus is too hacky. I'm suggesting a hybrid of chef/omnibus-software#511 and this PR.